src/lib/marshal/workflow/helpers.ts

```diff
@@ -104,6 +110,8 @@ const batchStepSummaryLines = (step: WorkflowStepData) => {
     batch_key && `Batch key: ${batch_key}`,
     duration && `Batch window: ${duration.value} ${duration.unit}`,
     field_path && `Batch window: "${field_path}"`,
+    batch_window_type &&
```

Comment:
Nit: Pretty sure this `batch_window_type &&` is not meaningful because we always have a default.

Reply:
Got carried away with the checks 😅